docs/examples/perceptualdecisionmaking.ipynb

```diff
@@ -0,0 +1,434 @@
+   "source": [
+    "# 1. Training an Agent on the Perceptual Decision Making Task\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 1.1 Environment Setup and Initial Agent Behavior\n",
+    "\n",
+    "Let's now create and explore the environment using the `PerceptualDecisionMaking` class from neurogym. We'll use the default configuration which includes:\n",
+    "\n",
+    "- `dim_ring = 2`: Two possible choices (left/right) represented at 0° and 180°. Note that the ring architecture can support any number of choices, making it suitable for more complex decision-making scenarios.\n",
+    "- `timing = {'fixation': ~600, 'stimulus': 2000, 'delay': 0, 'decision': 100}` (in milliseconds).\n",
+    "- `rewards = {'abort': -0.1, 'correct': +1.0, 'fail': 0.0}`; abort is a penalty applied when the agent fails to fixate. The task allows the trial to be aborted if fixation does not occur, which is where the name of this penalty comes from.\n",
+    "- `sigma = 1.0`: Standard deviation of the noise added to the inputs.\n",
+    "\n",
+    "In this notebook, several wrappers are used to modify the environment's behavior:\n",
+    "\n",
+    "- `ReactionTime` wrapper allows the agent to respond at any time after stimulus onset.\n",
+    "- `SideBias` wrapper introduces blocks where one side is more likely than the other. It uses two key parameters:\n",
+    "  - `probs = [[0.2, 0.8], [0.8, 0.2]]`: Probability matrices defining the likelihood of each choice (only two choices in this example) in different blocks\n",
+    "  - `block_dur = (20, 100)`: Block duration randomly sampled between 20-100 trials, determining how long each bias condition persists\n",
+    "- `PassAction` and `PassReward` wrappers augment the observations with the previous step's action and reward, respectively, enabling the agent to use recent history in decision-making.\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 1.1.1 Import Libraries\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import warnings\n",
+    "import matplotlib.pyplot as plt\n",
+    "import numpy as np\n",
+    "import neurogym as ngym\n",
+    "import gymnasium as gym\n",
+    "from neurogym.wrappers.monitor import Monitor\n",
+    "from sb3_contrib import RecurrentPPO\n",
+    "from stable_baselines3.common.vec_env import DummyVecEnv\n",
+    "from pathlib import Path\n",
+    "from neurogym.utils.ngym_random import TruncExp\n",
+    "from neurogym.wrappers.reaction_time import ReactionTime\n",
+    "from neurogym.wrappers.pass_action import PassAction\n",
+    "from neurogym.wrappers.pass_reward import PassReward\n",
+    "from neurogym.wrappers.side_bias import SideBias\n",
+    "from neurogym.utils.psychometric import plot_psychometric\n",
+    "\n",
+    "# Suppress warnings\n",
+    "warnings.filterwarnings(\"ignore\")"
```

Comment:
since #207, the first import cell generates a bunch of warnings from tensorflow. Add these lines to clear them to avoid confusing/worrying the user:

```suggestion
    "# Suppress warnings\n",
    "warnings.filterwarnings(\"ignore\")"
    "from IPython.display import clear_output\n",
    "clear_output()\n",
```